src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/versionMatcher/versionObjectDpkg.cpp

```diff
@@ -0,0 +1,14 @@
+/*
+ * Wazuh Vulnerability scanner - Database Feed Manager
+ * Copyright (C) 2015, Wazuh Inc.
+ * December 06, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "versionObjectDpkg.hpp"
+
+std::regex VersionObjectDpkg::m_parserRegex(R"(^(\d+)?:?(\d[\w\.+~\-]*)\-([\w\.+~]*)$)");
```

Comment:
I think, this regex is not correct.
I attach an image of strings that matches it but misrecognizes the epoch part.
![image](https://github.com/wazuh/wazuh/assets/101227434/444a6d1d-07ee-4067-b254-d6c2f43d6b82)

Reply:
Removed, we change the implementation :)